code_climate.rb

```diff
@@ -0,0 +1,175 @@
+require "net/http"
+require "json"
+require "active_support/core_ext"
+
+CODECLIMATE_API_TOKEN = "your CodeClimage API token"
```

Comment:
Typo.  Even so, I would just enforce this as an ENV var...prevents changing the code and thus accidentally committing the code.